install.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/sh
+set -e
+
+# Usage:
+# ENV_VAR= ./install.sh
+#
+# Environment variables:
+#
+# - ROLES="controlplane,node"
+#   Option by default, but controlplane role can be run standalone.
+
+# Function to get Linux distribution
+get_distro() {
+    DISTRO=$(egrep '^(ID)=' /etc/os-release| sed 's/"//g' | cut -f2 -d"=")
+}
+
+# Function to get system architecture
+get_arch() {
+    ARCH=$(uname -m)
+}
+
+# If RHEL, use subscription manager to register
+register_subs() {
+    subscription-manager register --auto-attach
+    subscription-manager repos --enable=rhocp-4.7-for-rhel-8-x86_64-rpms
+}
+
+# Apply SElinux policies (Permissive mode at the moment)
+apply_selinux_policy() {
+    setenforce 0
```

Comment:
https://github.com/redhat-et/microshift/issues/85#issuecomment-860981460

Reply:
Thanks @rootfs , I've updated the install script with these instructions.